.circleci/config.yml

```diff
@@ -221,4 +232,4 @@ jobs:
             pytest --cov=./ tests
             codecov
           environment:
-            OMP_NUM_THREADS: 1
+            OMP_NUM_THREADS: 1
```

Comment:
Seems like unrelated changes. Could you for instance merge the latest master branch to get rid of those?